src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -53,7 +53,7 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
         const uartHardware_t *hardware = &uartHardware[hindex];
         const UARTDevice_e device = hardware->device;
 
-#if !(defined(STM32F1) || defined(STM32F4)) // Older CPUs don't support pin swap.
+#if !defined(STM32F4) // Older CPUs don't support pin swap.
```

Comment:
`Older CPU`

Reply:
Changed to simply `// Don't support pin swap.`